src/components/Navbar.jsx

```diff
@@ -32,11 +32,22 @@ const Navbar = () => {
               <div className="ml-auto">
                 <div className="hidden md:flex md:space-x-2">
                   <NavLink to="/" className={`text-white hover:bg-gray-900 rounded-md px-3 py-2 ${isActive('/') && 'bg-black'}`}>Home</NavLink>
-                  <NavLink to="/absolute/STV" 
-                    className={`text-white hover:bg-gray-900 rounded-md px-3 py-2 ${(isActive('/absolute/STV') || isActive('/absolute/CTP') || isActive('/absolute/MW') || isActive('/absolute/KTG')) && 'bg-black'}`}>Absolute</NavLink>
-                  <NavLink to="/ally" className={`text-white hover:bg-gray-900 rounded-md px-3 py-2 ${isActive('/ally') && 'bg-black'}`}>Ally</NavLink>
-                  {/*<NavLink to="/revo/TP" 
-                    className={`text-white hover:bg-gray-900 rounded-md px-3 py-2 ${(isActive('/revo/TP') || isActive('/revo/Orchard') || isActive('/revo/Bugis') || isActive('/revo/Suntec')) && 'bg-black'}`}>Revo</NavLink>*/}
+                  {/* Studios Dropdown */}
```

Comment:
this is a good idea, although more testing is needed
when I'm on the Absolute page,
- studio should have a black background
- mouse over "Ally" has unexpected background colour changes